### PR TITLE
Create admin_news: show list news

### PR DESCRIPTION
**Pull request by tinhdang**
Create NewsController : show list news

### CODE REVIEW

File: app/Http/Controllers/Admin/NewsController.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     //* @param  \Illuminate\Http\Request $request
```
Comment (ghost) [inline app/Http/Controllers/Admin/NewsController.php]: if you don't use this code, pls remove, don't comment out

File: resources/views/admin/news/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <th style="width:8%">{{trans('news.user')}}</th>
+            <th>{{trans('news.content')}}</th>
+            <th>{{trans('news.image')}}</th>
+            <th>{{trans('admin.action')}}</th>
+          </tr>
+          @foreach($news as $item)
+          <tr>
+            <td>{{$item->id}}</td>
+            <td>{{$item->title}}</td>
+            <td>{{$item->category->name}}</td>
```
Comment (at-vienle) [inline resources/views/admin/news/index.blade.php]: this code have n+1 problem. Pls use Laravel Eager loading
https://laravel.com/docs/5.4/eloquent-relationships#eager-loading

File: resources/views/admin/news/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        </div>
+      </div>
+      <div class="box-body table-responsive no-padding">
+        <table class="table table-hover">
+          <tr>
+            <th>#</th>
+            <th style="width:15%">{{trans('news.title')}}</th>
+            <th style="width:8%">{{trans('categories.category')}}</th>
+            <th style="width:8%">{{trans('news.user')}}</th>
+            <th>{{trans('news.content')}}</th>
```
Comment (ghost) [inline resources/views/admin/news/index.blade.php]: I think we don't need to show content, and image on the list

**Review by ghost — CHANGES_REQUESTED** [review]
pls check my cmt

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**